### ISSUE TITLE
Directly show html version of our documentation

### ISSUE THREAD

**Issue by bgeuken** [openSUSE/obs-landing#95]
When going to the [help](http://openbuildservice.org/help/) page users are offered 3 different presentation formats, html, pdf and epub.

I think it would be better to show the html version directly. The other formats could be offered via a decent icon.

**Comment by hennevogel** [comment]
But there is also the choice of different "books"...

**Comment by bgeuken** [comment]
Yes, it would also be nice to (interlink)[https://github.com/openSUSE/obs-docu/issues/88] the guides:-) Right now one needs to many clicks to get to the actual docu
